package.json

```diff
@@ -1,21 +1,10 @@
 {
-  "name": "tech-doc-hugo",
-  "version": "0.0.1",
-  "description": "Hugo theme for technical documentation.",
-  "main": "none.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/bep/tech-doc-hugo.git"
-  },
-  "author": "",
-  "license": "ISC",
-  "bugs": {
-    "url": "https://github.com/bep/tech-doc-hugo/issues"
-  },
-  "homepage": "https://github.com/bep/tech-doc-hugo#readme",
+  "name": "tetra",
+  "version": "0.1.0",
+  "description": "Project Tetra",
+  "repository": "https://github.com/tetrabiodistributed/tetra.git",
+  "license": "MIT",
```

Comment:
Wait, are we MIT or CERN?

Reply:
Documentation is MIT. STL files are CERN